src/Adapter/MSTest.TestAdapter/Execution/TcmTestPropertiesProvider.cs

```diff
@@ -1,50 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Wrongly detected as removed + added, it was moved.

Reply:
I think it's seen that way due to the Constants -> EngineConstants change which affects many lines. But that's fine 👍